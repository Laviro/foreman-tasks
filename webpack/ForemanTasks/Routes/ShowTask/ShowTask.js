import React from 'react';

const ShowTask = () => (
  <div>
    <h1>Hello Foreman Tasks</h1>
    <h2>show-task-page</h2>
  </div>
);

export default ShowTask;
