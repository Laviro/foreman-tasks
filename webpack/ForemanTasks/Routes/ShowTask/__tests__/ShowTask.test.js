import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';

import ShowTask from '../ShowTask';

const fixtures = {
  'render without Props': {},
};

describe('ShowTask', () =>
  testComponentSnapshotsWithFixtures(ShowTask, fixtures));
