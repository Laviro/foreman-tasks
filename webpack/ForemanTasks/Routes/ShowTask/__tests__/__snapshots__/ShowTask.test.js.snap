// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowTask render without Props 1`] = `
<div>
  <h1>
    Hello Foreman Tasks
  </h1>
  <h2>
    show-task-page
  </h2>
</div>
`;
